lib/xapi/docs.rb

```diff
@@ -0,0 +1,49 @@
+module Xapi
+  # Some docs
+  class Docs
+    AUTHORIZED_DOCS = [
+      "HELLO.md",
+      "WORKFLOW.md",
+      "RESOURCES.md",
+      "INSTALLATION.md",
+      "TOOLS.md",
+    ]
```

Comment:
I think it would be useful to have the keys be lowercase: `hello`, `workflow`, `resources`, `installation`, and `tools`.